pkg/machine/e2e/machine_test.go

```diff
@@ -11,8 +11,8 @@ import (
 	"testing"
 	"time"
 
+	pf "github.com/containers/podman/v4/cmd/podman/machine"
```

Comment:
do not import cmd/.... from pkg/

see https://github.com/containers/podman/pull/14850 for more info why this is a bad idea.